### PR TITLE
Start adding commercial docs to mainline (SYN-2954)

### PR DESCRIPTION
**Pull request by vEpiphyte**

### CODE REVIEW

File: docs/synapse/power_ups.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+manage YARA rules, scans, and matches.
+
+For a introduction to Power-Ups from our analysts and seeing them in use, see the following video introducing them:
+
+.. raw:: html
+
+   <iframe width="560" height="315" src="https://www.youtube.com/embed/eb6wEYGRTyY" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+Vertex Project is constantly releasing new Power-Ups and expanding features of existing Power-Ups. If you join the
+``#synapse-release`` channel in Synapse Slack_, you can get realtime notices of these updates!
```
Comment (Cisphyx) [inline docs/synapse/power_ups.rst:19]: missing s on ``#synapse-releases``

File: docs/synapse/devguides/cortex.hcl [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        # /data/vertex on the host node
+        volumes = [
+          "/data/vertex/synapse_core01/:/vertex/storage",
+        ]
+
+        # Assign names to the telepath and https API ports
+        # for later use.
+        port_map {
+          telepath = 27492
+          https = 4433
```
Comment (Cisphyx) [inline docs/synapse/devguides/cortex.hcl:28]: should this be 4443?

File: docs/synapse/devguides/orch_hashistack.rst [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+A second example is shown below, this time for the Synapse Shodan Connector, a Storm Service. This example differs
+mainly in that it shows how to use a ``cell.yaml`` file to do some boot time configuration.
+
+- First, a ``template`` directive is used to push a YAML file to ``./local/cell.yaml`` when the job is deployed.
+- Second, the Docker entrypoint has been replaced with shell script one-liner. This copies the data from
+  ``./local/cell.yaml`` and over to ``/vertex/storage/cell.yaml``. This location would be the mapped in persistent
+  storage location for the service.
+- Last, after copying the file, the shell script launches the service process.
+- In addition, this example also shows a place where a authentication username for Docker Hub (our container image
```
Comment (Cisphyx) [inline docs/synapse/devguides/orch_hashistack.rst:39]: `an` authentication username

File: docs/synapse/devguides/orch_hashistack.rst [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+the following example::
+
+    tcp+consul://root:secret@synapse-core01/*?consul=http://consul.vpc.lan:8500&consul_tag=telepath
+
+    # Invoking command from the command line
+    python -m synapse.tools.cmdr "tcp+consul://root:secret@synapse-core01/*?consul=http://consul.vpc.lan:8500&consul_tag=telepath"
+
+From this, the Telepath Proxy looks up the ``synapse-core01`` service in the Consul catalog, and retrieves the first
+entry from that service which matches the tag ``telepath``. It then uses the IP and Port from the catalog entry in order
+to connect to the Cortex. Another example would be using this to connect the Shodan service to the cortex::
```
Comment (Cisphyx) [inline docs/synapse/devguides/orch_hashistack.rst:63]: got a lowercase "c" in Cortex

File: docs/synapse/devguides/orch_kubernetes.rst [hunk 5]
```diff
@@ -0,0 +1,80 @@
+.. _orch-kubernetes:
+
+Kubernetes
+==========
+
+A popular option for Orchestration is Kubernetes. Kubernetes is an open-source system for automating the deployment,
+scaling and managemetn of containerized applications. We provide a examples that you can use to quickly get started
```
Comment (Cisphyx) [inline docs/synapse/devguides/orch_kubernetes.rst:7]: typo "mangemetn"

Comment (Cisphyx) [reply docs/synapse/devguides/orch_kubernetes.rst:7]: "provide a examples"

File: docs/synapse/devguides/orch_kubernetes.rst [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+using Kubernetes to orchestrate your Synapse deployment.  These examples include an Aha cell, a Axon, a Cortex,
+the Maxmind connector, and the Optic UI.
+
+Since all Telepath services connect via Aha, this allows for easy lookup of services via Aha. This allows for users to
+ignore most application awareness of port numbers. For example, the Maxmind connector can easily be be added to the
```
Comment (Cisphyx) [inline docs/synapse/devguides/orch_kubernetes.rst:12]: "be be"

File: docs/synapse/devguides/orch_kubernetes.rst [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+Cortex via ``service.add maxmind aha://root:demo@maxmind.aha.demo.net``.
+
+The Optic deployment uses a ``initContainers`` container to copy the TLS certificates into the service directory for
```
Comment (Cisphyx) [inline docs/synapse/devguides/orch_kubernetes.rst:15]: "an" initContainers

File: docs/synapse/devguides/orch_kubernetes.rst [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+Optic. The Traefik ``IngressRouteTCP`` directs all TLS traffic to the service to the Optic service. Since the TLS
+certificates have been put into the Cell directory for Optic, and the ``IngressRouteTCP`` acts a TLS passthrough,
+users are using TLS end to end to connect to Optic.
+
+Passwords used for doing inter-service communications are stored in Kubernetes Secrets and are interpolated from
+environment variables form Telepath URLs when needed. To keep these examples from being too large, passwords are shared
```
Comment (Cisphyx) [inline docs/synapse/devguides/orch_kubernetes.rst:21]: sounds like there is a word missing somewhere in "interpolated from environment variables form Telepath URLs when needed"

File: docs/synapse/power_ups.rst [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+
+Synapse Power-ups
+#################
+
+
+Power-Ups provide specific add-on capabilities to Synapse via Storm Packages and Services. For example, Power-Ups may
+provide connectivity to external databases, third-party data sources, or enable functionality such as the ability to
+manage YARA rules, scans, and matches.
+
+For a introduction to Power-Ups from our analysts and seeing them in use, see the following video introducing them:
```
Comment (Cisphyx) [inline docs/synapse/power_ups.rst:12]: "an" introduction

File: docs/synapse/power_ups/rapid_power_ups.rst [hunk 10]
```diff
@@ -0,0 +1,97 @@
+.. _rapid-powerups:
+
+Rapid Power-Ups
+###############
+
+Rapid Power-ups are designed to delivered to a Cortex as Storm packages. This allows users to rapidly expand the power
```
Comment (Cisphyx) [inline docs/synapse/power_ups/rapid_power_ups.rst:6]: "designed to delivered"

File: docs/synapse/power_ups/rapid_power_ups.rst [hunk 11]
```diff
@@ -0,0 +1,97 @@
...
+of their deployments without needing to deploy additional containers to their environments. For a introduction to
```
Comment (Cisphyx) [inline docs/synapse/power_ups/rapid_power_ups.rst:7]: "an" introduction

File: docs/synapse/power_ups/rapid_power_ups.rst [hunk 12]
```diff
@@ -0,0 +1,97 @@
...
+
+Getting Started with Rapid Power-Ups
+------------------------------------
+
+Vertex maintains a package repository which allows for loading public and private packages.
+
+If you are a :ref:`synapse-ui` user, you can navigate to the Power-Ups tab to register your Cortex and configure packages
+directly from the UI.
+
+Alternatively, one can use the `storm`_ tool can also be used.
```
Comment (Cisphyx) [inline docs/synapse/power_ups/rapid_power_ups.rst:56]: extra words in  "one can use the `storm`_ tool can also be used."

File: docs/synapse/user_interface.rst [hunk 13]
```diff
@@ -0,0 +1,34 @@
...
+======================
+
+
+The Synapse UI is a WebUI which integrates with a Cortex to provide a view into your Synapse
+Ecosystem. A few important UI features include:
+
+    - Directly supports collaborative analysis. Users can view changes made by their teammates immediately. This enables
+      them to rapidly work together as peers.
+
+    - Simplified Storm query bar and point-and-click navigation for exploring data. User can visualize their data in
```
Comment (Cisphyx) [inline docs/synapse/user_interface.rst:13]: Sounds like this should be "A user" or "Users"

File: docs/synapse/devguides/cortex.hcl (hunk 2 above)
Comment (vEpiphyte) [reply docs/synapse/devguides/cortex.hcl:28]: fixed

File: docs/synapse/devguides/orch_hashistack.rst (hunk 3 above)
Comment (vEpiphyte) [reply docs/synapse/devguides/orch_hashistack.rst:39]: fixed

File: docs/synapse/devguides/orch_hashistack.rst (hunk 4 above)
Comment (vEpiphyte) [reply docs/synapse/devguides/orch_hashistack.rst:63]: fixed

File: docs/synapse/devguides/orch_kubernetes.rst (hunk 5 above)
Comment (vEpiphyte) [reply docs/synapse/devguides/orch_kubernetes.rst:7]: fixed

File: docs/synapse/devguides/orch_kubernetes.rst [hunk 14]
```diff
@@ -0,0 +1,80 @@
+.. _orch-kubernetes:
+
+Kubernetes
+==========
+
+A popular option for Orchestration is Kubernetes. Kubernetes is an open-source system for automating the deployment,
+scaling and managemetn of containerized applications. We provide a examples that you can use to quickly get started
+using Kubernetes to orchestrate your Synapse deployment.  These examples include an Aha cell, a Axon, a Cortex,
```
[the remark this answers is not in the document]
Comment (vEpiphyte) [reply docs/synapse/devguides/orch_kubernetes.rst:8]: fixed

File: docs/synapse/devguides/orch_kubernetes.rst (hunk 6 above)
Comment (vEpiphyte) [reply docs/synapse/devguides/orch_kubernetes.rst:12]: fixed

File: docs/synapse/devguides/orch_kubernetes.rst (hunk 7 above)
Comment (vEpiphyte) [reply docs/synapse/devguides/orch_kubernetes.rst:15]: fixed

File: docs/synapse/power_ups.rst (hunk 9 above)
Comment (vEpiphyte) [reply docs/synapse/power_ups.rst:12]: fixed

File: docs/synapse/power_ups/rapid_power_ups.rst (hunk 10 above)
Comment (vEpiphyte) [reply docs/synapse/power_ups/rapid_power_ups.rst:6]: fixed / rewritten

File: docs/synapse/power_ups/rapid_power_ups.rst (hunk 11 above)
Comment (vEpiphyte) [reply docs/synapse/power_ups/rapid_power_ups.rst:7]: fixed

File: docs/synapse/power_ups/rapid_power_ups.rst (hunk 12 above)
Comment (vEpiphyte) [reply docs/synapse/power_ups/rapid_power_ups.rst:56]: fixed

File: docs/synapse/user_interface.rst (hunk 13 above)
Comment (vEpiphyte) [reply docs/synapse/user_interface.rst:13]: fixed

File: docs/synapse/devguides/orch_kubernetes.rst (hunk 8 above)
Comment (vEpiphyte) [reply docs/synapse/devguides/orch_kubernetes.rst]: fixed!

**Review by Cisphyx — APPROVED** [review]

**Pull request merged**